### PR TITLE
Cloud Run Eventing

### PR DESCRIPTION
**Pull request by TheJaySmith**
Demo on using Cloud Run for Anthos with Knative Eventing and Kafka from Confluent Cloud to do Severless Event Streaming

https://cloudonair.withgoogle.com/events/partner-up-confluent

### CODE REVIEW

**Comment by ToddKopriva** [comment]
Sorry for the delay on getting to this review. I'll review it this week.

**Comment by TheJaySmith** [comment]
Howdy, Just checking in

**Comment by ToddKopriva** [comment]
I apologize again for the delay. We're in the middle of upgrading our publishing pipeline, which has caused some work items to jump the queue. Your tutorial is next in line.

File: tutorials/cloud-run-eventing-kafka/kafka-cr-eventing/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (ToddKopriva) [inline tutorials/cloud-run-eventing-kafka/kafka-cr-eventing/LICENSE]: Please remove this license file.

See this license at the top level of this repository, which applies the Apache 2.0 license to all code samples in this repository:
https://github.com/GoogleCloudPlatform/community/blob/master/LICENSE.md

**Comment by ToddKopriva** [comment]
I'll publish this to cloud.google.com/community tomorrow.

**Comment by ToddKopriva** [comment]
published:
https://cloud.google.com/community/tutorials/cloud-run-eventing-kafka

**Pull request merged**